src/console-services.yml

```diff
@@ -99,57 +175,125 @@
   description: 'EC2 - Virtual Servers in the Cloud'
   url: 'https://console.aws.amazon.com/ec2/home'
 
+- command: ecr
+  description: 'ECR - Fully-managed Docker container registry'
+  url: 'https://console.aws.amazon.com/ecr/home'
+
 - command: ecs
-  description: 'EC2 Container Service - Run and Manage Docker Containers'
+  description: 'ECS - Run and Manage Docker Containers'
```

Comment:
we should have the expanded name in the description. i see that there are some old names that don't follow this, but we should attempt to do this going forward to provide more definition to what the search term. adding the abbreviation after the term (e.g., `Elastic Container Service (ECS)`) could be useful too if it's more well known as its abbreviation

Reply:
This is the description in the Console so I just followed suit. It's is actually called Amazon Elastic Container Service, not EC2 Container Service.

Reply:
k sounds good. yeah, EC2 Container Service was [its original name](https://aws.amazon.com/about-aws/whats-new/2014/11/13/introducing-amazon-ec2-container-service/). it seems to have been rebranded since then